From 1fe7b756365254f7175558e7a89c1d89f95c5e2e Mon Sep 17 00:00:00 2001
From: Maxime Hadjinlian <maxime.hadjinlian@devialet.com>
Date: Tue, 30 Sep 2014 13:46:59 +0200
Subject: [PATCH] Remove OWNER and GROUPS parameters to install

The Install tries to change the owner and groups of the binary. In
Buildroot we don't want that, since fakeroot will do its job when we
finalize the rootfs.

Signed-off-by: Maxime Hadjinlian <maxime.hadjinlian@devialet.com>
[aduskett@gmail.com: Updated for 32408520fcebe785983a68e39ec83830a3005779]
Signed-off-by: Adam Duskett <Aduskett@gmail.com>
---
 docbook/Makefile | 4 ++--
 ether/Makefile   | 8 ++++----
 key/Makefile     | 8 ++++----
 mdio/Makefile    | 8 ++++----
 mme/Makefile     | 8 ++++----
 nvm/Makefile     | 8 ++++----
 pib/Makefile     | 8 ++++----
 plc/Makefile     | 8 ++++----
 ram/Makefile     | 8 ++++----
 serial/Makefile  | 8 ++++----
 slac/Makefile    | 8 ++++----
 tools/Makefile   | 2 +-
 12 files changed, 43 insertions(+), 43 deletions(-)

diff --git a/docbook/Makefile b/docbook/Makefile
index d630822..039d62a 100755
--- a/docbook/Makefile
+++ b/docbook/Makefile
@@ -32,9 +32,9 @@ manuals:
 	${SHELL} ../docbook/manuals
 scripts:
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${LIBRARY}
+	install -m ${DIR_PERM} -d ${LIBRARY}
 install: library
-	if [ -d ${LIBRARY} ]; then install -m ${WWW_PERM} -o ${OWNER} -g ${GROUP} *.html *.css *.png ${LIBRARY}; fi
+	if [ -d ${LIBRARY} ]; then install -m ${WWW_PERM} *.html *.css *.png ${LIBRARY}; fi
 uninstall:
 	rm -f ${LIBRARY}/*.c.html ${LIBRARY}/*.h.html ${LIBRARY}/*.1.html
 setup:
diff --git a/ether/Makefile b/ether/Makefile
index 89174cb..8256d50 100755
--- a/ether/Makefile
+++ b/ether/Makefile
@@ -27,13 +27,13 @@ compact: compile
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${BIN}
+	install -m ${DIR_PERM} -d ${BIN}
 scripts:
 manuals:
-	test -d ${MAN} || install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${MAN}
-	install -m ${MAN_PERM} -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}
+	test -d ${MAN} || install -m ${DIR_PERM} -d ${MAN}
+	install -m ${MAN_PERM} ${PAGES} ${MAN}
 install: $(TOOLS) library
-	install -m ${SUID_PERM} -o ${OWNER} -g ${GROUP} ${TOOLS} ${BIN}
+	install -m ${SUID_PERM} ${TOOLS} ${BIN}
 uninstall:
 	cd ${BIN}; rm -f ${TOOLS}
 	cd ${MAN}; rm -f ${PAGES}
diff --git a/key/Makefile b/key/Makefile
index de62a49..78fa200 100755
--- a/key/Makefile
+++ b/key/Makefile
@@ -28,13 +28,13 @@ compact: compile
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${BIN}
+	install -m ${DIR_PERM} -d ${BIN}
 scripts:
 manuals:
-	test -d ${MAN} || install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${MAN}
-	install -m ${MAN_PERM} -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}
+	test -d ${MAN} || install -m ${DIR_PERM} -d ${MAN}
+	install -m ${MAN_PERM} ${PAGES} ${MAN}
 install: $(TOOLS)
-	install -m ${BIN_PERM} -o ${OWNER} -g ${GROUP} ${TOOLS} ${BIN}
+	install -m ${BIN_PERM} ${TOOLS} ${BIN}
 uninstall:
 	cd ${BIN}; rm -f ${TOOLS}
 	cd ${MAN}; rm -f ${PAGES}
diff --git a/mdio/Makefile b/mdio/Makefile
index 049b08e..3ff4c26 100755
--- a/mdio/Makefile
+++ b/mdio/Makefile
@@ -27,13 +27,13 @@ compact: compile
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${BIN}
+	install -m ${DIR_PERM} -d ${BIN}
 scripts:
 manuals:
-	test -d ${MAN} || install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${MAN}
-	install -m ${MAN_PERM} -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}
+	test -d ${MAN} || install -m ${DIR_PERM} -d ${MAN}
+	install -m ${MAN_PERM} ${PAGES} ${MAN}
 install: $(TOOLS) library
-	install -m ${BIN_PERM} -o ${OWNER} -g ${GROUP} ${TOOLS} ${BIN}
+	install -m ${BIN_PERM} ${TOOLS} ${BIN}
 uninstall:
 	cd ${BIN}; rm -f int6k ${TOOLS}
 	cd ${MAN}; rm -f int6k ${PAGES}
diff --git a/mme/Makefile b/mme/Makefile
index 0313aea..77a6b96 100755
--- a/mme/Makefile
+++ b/mme/Makefile
@@ -27,13 +27,13 @@ compact: compile
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${BIN}
+	install -m ${DIR_PERM} -d ${BIN}
 scripts:
 manuals:
-	test -d ${MAN} || install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${MAN}
-	install -m ${MAN_PERM} -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}
+	test -d ${MAN} || install -m ${DIR_PERM} -d ${MAN}
+	install -m ${MAN_PERM} ${PAGES} ${MAN}
 install: $(TOOLS) library
-	install -m ${BIN_PERM} -o ${OWNER} -g ${GROUP} ${TOOLS} ${BIN}
+	install -m ${BIN_PERM} ${TOOLS} ${BIN}
 uninstall:
 	cd ${BIN}; rm -f ${TOOLS}
 	cd ${MAN}; rm -f ${PAGES}
diff --git a/nvm/Makefile b/nvm/Makefile
index 9aa05ec..b468e70 100755
--- a/nvm/Makefile
+++ b/nvm/Makefile
@@ -28,12 +28,12 @@ compact: compile
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 scripts:
 manuals:
-	test -d ${MAN} || install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${MAN}
-	install -m ${MAN_PERM} -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}
+	test -d ${MAN} || install -m ${DIR_PERM} -d ${MAN}
+	install -m ${MAN_PERM} ${PAGES} ${MAN}
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${BIN}
+	install -m ${DIR_PERM} -d ${BIN}
 install: $(TOOLS) library
-	install -m ${BIN_PERM} -o ${OWNER} -g ${GROUP} ${TOOLS} ${BIN}
+	install -m ${BIN_PERM} ${TOOLS} ${BIN}
 uninstall:
 	cd ${BIN}; rm -f ${TOOLS}
 	cd ${MAN}; rm -f ${PAGES}
diff --git a/pib/Makefile b/pib/Makefile
index 93f0c3b..23195b4 100644
--- a/pib/Makefile
+++ b/pib/Makefile
@@ -30,12 +30,12 @@ compact: compile
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 scripts:
 manuals:
-	test -d ${MAN} || install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${MAN}
-	install -m ${MAN_PERM} -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}
+	test -d ${MAN} || install -m ${DIR_PERM} -d ${MAN}
+	install -m ${MAN_PERM} ${PAGES} ${MAN}
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${BIN}
+	install -m ${DIR_PERM} -d ${BIN}
 install: $(TOOLS) library
-	install -m ${BIN_PERM} -o ${OWNER} -g ${GROUP} ${TOOLS} ${BIN}
+	install -m ${BIN_PERM} ${TOOLS} ${BIN}
 uninstall:
 	cd ${BIN}; rm -f ${TOOLS}
 	cd ${MAN}; rm -f ${PAGES}
diff --git a/plc/Makefile b/plc/Makefile
index 43d6bd5..9fd4e0a 100644
--- a/plc/Makefile
+++ b/plc/Makefile
@@ -32,13 +32,13 @@ compact: compile
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${BIN}
+	install -m ${DIR_PERM} -d ${BIN}
 scripts:
 manuals:
-	test -d ${MAN} || install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${MAN}
-	install -m ${MAN_PERM} -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}
+	test -d ${MAN} || install -m ${DIR_PERM} -d ${MAN}
+	install -m ${MAN_PERM} ${PAGES} ${MAN}
 install: $(TOOLS) library
-	install -m ${SUID_PERM} -o ${OWNER} -g ${GROUP} ${TOOLS} ${BIN}
+	install -m ${SUID_PERM} ${TOOLS} ${BIN}
 uninstall:
 	cd ${BIN}; rm -f ${TOOLS}
 	cd ${MAN}; rm -f ${PAGES}
diff --git a/ram/Makefile b/ram/Makefile
index 60b9bb3..c34ddd0 100755
--- a/ram/Makefile
+++ b/ram/Makefile
@@ -27,13 +27,13 @@ compact: compile
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${BIN}
+	install -m ${DIR_PERM} -d ${BIN}
 scripts:
 manuals:
-	test -d ${MAN} || install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${MAN}
-	install -m ${MAN_PERM} -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}
+	test -d ${MAN} || install -m ${DIR_PERM} -d ${MAN}
+	install -m ${MAN_PERM} ${PAGES} ${MAN}
 install: $(TOOLS) library
-	install -m ${BIN_PERM} -o ${OWNER} -g ${GROUP} ${TOOLS} ${BIN}
+	install -m ${BIN_PERM} ${TOOLS} ${BIN}
 uninstall:
 	cd ${BIN}; rm -f ${TOOLS}
 	cd ${MAN}; rm -f ${PAGES}
diff --git a/serial/Makefile b/serial/Makefile
index c25f1a6..aebbac9 100755
--- a/serial/Makefile
+++ b/serial/Makefile
@@ -27,13 +27,13 @@ compact: compile
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${BIN}
+	install -m ${DIR_PERM} -d ${BIN}
 scripts:
 manuals:
-	test -d ${MAN} || install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${MAN}
-	install -m ${MAN_PERM} -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}
+	test -d ${MAN} || install -m ${DIR_PERM} -d ${MAN}
+	install -m ${MAN_PERM} ${PAGES} ${MAN}
 install: $(TOOLS) library
-	install -m ${BIN_PERM} -o ${OWNER} -g ${GROUP} ${TOOLS} ${BIN}
+	install -m ${BIN_PERM} ${TOOLS} ${BIN}
 uninstall:
 	cd ${BIN}; rm -f int6k ${TOOLS}
 	cd ${MAN}; rm -f int6k ${PAGES}
diff --git a/slac/Makefile b/slac/Makefile
index bd12db9..1f0b65d 100755
--- a/slac/Makefile
+++ b/slac/Makefile
@@ -27,13 +27,13 @@ compact: compile
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${BIN}
+	install -m ${DIR_PERM} -d ${BIN}
 scripts:
 manuals:
-	if [ ! -d ${MAN} ]; then install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${MAN}; fi
-	install -m ${MAN_PERM} -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}
+	if [ ! -d ${MAN} ]; then install -m ${DIR_PERM} -d ${MAN}; fi
+	install -m ${MAN_PERM} ${PAGES} ${MAN}
 install: compile library
-	install -m ${SUID_PERM} -o ${OWNER} -g ${GROUP} ${TOOLS} ${BIN}
+	install -m ${SUID_PERM} ${TOOLS} ${BIN}
 uninstall:
 	cd ${BIN}; rm -f ${TOOLS}
 	cd ${MAN}; rm -f ${PAGES}
diff --git a/tools/Makefile b/tools/Makefile
index 1d992cf..a74fd24 100755
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -27,7 +27,7 @@ compact: compile
 	if [ ! -z ${STRIP} ]; then ${STRIP} ${TOOLS}; fi
 	if [ -x /usr/local/bin/upx ]; then upx --best ${TOOLS}; fi
 library:
-	install -m ${DIR_PERM} -o ${OWNER} -g ${GROUP} -d ${BIN}
+	install -m ${DIR_PERM} -d ${BIN}
 scripts:
 manuals:
 #	install -m ${MAN_PERM} -o ${OWNER} -g ${GROUP} ${PAGES} ${MAN}
-- 
2.26.2

